main.go

```diff
@@ -280,7 +280,7 @@ func defineFlagVar() *FlagVar {
 		"The resources.limits.cpu for skr webhook.")
 	flag.StringVar(&flagVar.virtualServiceName, "virtual-svc-name", "kcp-events",
 		"Name of the virtual service resource to be reconciled by the watcher control loop.")
-	flag.StringVar(&flagVar.virtualServiceName, "gateway-name", "lifecycle-manager-kyma-gateway",
+	flag.StringVar(&flagVar.gatewaySelector, "gateway-selector", "operator.kyma-project.io/gateway=default",
```

Comment:
Also here it should have values bound to v1alpha1

Reply:
Why is the gateway-selector hard-coded in the app? Wouldn't it make sense to put it in the Kyma/Watcher instead?

Reply:
`Wouldn't it make sense to put it in the Kyma/Watcher instead?` I am not sure if I understand... I defined a constant for it in the v1alpha1 package. Is that enough?

Reply:
I think what I meant is that the selector probably is not something we should define at operator-startup, but dynamically based on a setting in the Kyma or Watcher CR. WDYT?

Reply:
I did as you suggested, It's taken from the Watcher CR now, if defined.